eng/pipelines/coreclr/perf-wasm-prepare-artifacts-steps.yml

```diff
@@ -0,0 +1,15 @@
+parameters:
+  configForBuild: 'Release'
+
+steps:
+  - script: "./dotnet.sh build -p:TargetOS=Browser -p:TargetArchitecture=wasm -p:Configuration=${{ parameters.configForBuild }} /nr:false /p:TreatWarningsAsErrors=true /p:ContinuousIntegrationBuild=true src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj /t:InstallWorkloadUsingArtifacts"
```

Comment:
So this is the magic part that makes a workload from the dotnet-runtime source tree and installs it?